src/style.css

```diff
@@ -0,0 +1,104 @@
+@font-face {
+  font-family: Anevir;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: AnevirBold;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+body * {
+  /*stylelint-disable-next-line font-family-no-missing-generic-family-keyword*/
+  font-family: Anevir;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+body {
+  margin-top: 0;
+}
+
+form[data-qa="big"],
+form[data-qa="small"] {
+  margin-top: 20px;
+}
```

Comment:
Why do you select by attributes?

Reply:
I did so because I have to put them and I could not fit them Into my layout with higher profit ... in this case those tags are used not only for testing purpose but for styling.
Your comment I'll accept like warn flag and will use those attrs in direct way next time!